common/src/main/java/com/wynntils/models/marker/type/LocationSupplier.java

```diff
@@ -8,5 +8,5 @@
 
 @FunctionalInterface
 public interface LocationSupplier {
-    Location getLocation();
+    Location location();
```

Comment:
Why the rename? `get` signals that this is not a constant record, but can be dynamic

Reply:
Just revert `StaticLocationSupplier` so it will work. Don't overdo the intellij suggestions..

Reply:
I didn't know it changes the names of the interfaces